pytensor/tensor/slinalg.py

```diff
@@ -908,6 +908,102 @@ def solve_discrete_are(A, B, Q, R, enforce_Q_symmetric=False) -> TensorVariable:
     )
 
 
+def largest_common_dtype(tensors: typing.Sequence[TensorVariable]) -> np.dtype:
```

Comment:
This in an odd place for this function? Maybe make it a classmethod of the Op for the time being?

Wouldn't want people finding and importing it from here. Can also put it in a better place instead but that's more work

Reply:
I was waiting for a resolution on the other convo thread about whether it's even needed

Reply:
We still need to define a dtype for the output.

Reply:
I renamed it to `_largest_common_dtype` and it's not in `__all__` so if a user finds it kudos to him I guess